internal/controllers/machine/machine_controller_noderef.go

```diff
@@ -136,6 +136,7 @@ func (r *Reconciler) reconcileNode(ctx context.Context, cluster *clusterv1.Clust
 	// Do the remaining node health checks, then set the node health to true if all checks pass.
 	status, message := summarizeNodeConditions(node)
 	if status == corev1.ConditionFalse {
+		// TODO: There's a bug where the node healthy condition is never set to true.
```

Comment:
is this still TODO? is it tracked by an issue?

Reply:
It's not in the scope of this PR but I'll open an issue!

Reply:
Never mind, it fixes itself once you install Calico.